RobotCode/src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.ArmSubsytem;
+
+public class ArmCommand extends Command
+{
+	public ArmSubsytem arm;
+	public double targetAngle;
+
+	public ArmCommand()
+	{
+		this.arm = arm;
+		
+		targetAngle = ArmConstants.TARGET_ANGLE;
```

Comment:
This isn't a constant value because we can set the arm to different positions, you need to pass a position into the constructor and set targetAngle equal to that position.